config/c-code/tests.yml.j2

```diff
@@ -151,7 +152,7 @@ jobs:
       - name: Install %(package_name)s
         run: |
           pip install -U wheel setuptools
-          pip install -U coverage
+          pip install -U --no-binary :all: coverage
```

Comment:
A comment explaining the `--no-binary :all:` would be helpful here so people don't have to dig through git logs.

Reply:
I added a comment in https://github.com/zopefoundation/meta/pull/132/commits/45b1f148eda3f6f46bde852104f8e06c4b3df267